nextjs-nabil/src/components/TablePlayers.js

```diff
@@ -0,0 +1,73 @@
+import { useContext } from "react";
+import { AppContext } from "../context";
+import Table from "@mui/material/Table";
```

Comment:
instead of creating separate imports you can just have it all import in this manner

import { Table, TableBody, etc } from "@mui/material/";

Reply:
Noted